### PR TITLE
Prevent Dialog Box

### LINKED ISSUE #443: BUG - Sometimes when I drag a library on the create run page it opens a dialog box
**Describe the bug**
If I drag a libray and select the tube picture it opens a dialog box asking me to save the picture

**RT Ticket Number**
If applicable

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to page ONT > Runs > New
2. Select and dag tube
3. Dialog box opens asking me if I want to save the picture

**Expected behaviour**
Nothing

**Screenshots**

**Desktop (please complete the following information):**
* Windows or Mac?
* Browser Chrome, Firefox, Safari or other?
* Browser version (use 'About' to look up)?

**Additional context**

### PR DESCRIPTION
**Pull request by Chris-Friend**
Closes #443 

Changes proposed in this pull request:

* Prevent the default drop action so in Firefox the tube image is not downloaded

### CODE REVIEW

File: src/components/ont/OntTube.vue [hunk 1]
```diff
@@ -16,6 +16,9 @@
 
 <script>
 
+const img = new Image()
+img.src = '/tube.png'
+
```
Comment (sdjm-tessella) [inline src/components/ont/OntTube.vue:21]: What did changing the scope of this constant achieve?  Making it truly constant?

Comment (Chris-Friend) [reply src/components/ont/OntTube.vue:21]: I noticed that if I didn't call `setDragImage` the drag was working as expected in Safari. After some digging I found [this](https://stackoverflow.com/a/22610927) slightly outdated StackOverflow, which did the trick.

I also noticed that if you kept defining the image in the event handler, on Safari it got added as a Source file each time (not sure if just a reference to the same image, a browser cache, or something else).

Pre-defining it and increasing it's scope fixed both of these issues

Comment (Chris-Friend) [reply src/components/ont/OntTube.vue:21]: As per the context of the change: the drag wasn't working in Safari

Comment (stevieing) [reply src/components/ont/OntTube.vue:21]: Strangely enough it was already like this in pacbio tube. As we are doing the same thing.

Comment (Chris-Friend) [reply src/components/ont/OntTube.vue:21]: Could've saved myself some time if I'd looked at that 😆

Comment (stevieing) [reply src/components/ont/OntTube.vue:21]: My fault. Should have pointed you to it. I assumed that the same drag and drop had been used. We need to abstract it into the same place as they do pretty much exactly the same thing.

**Pull request merged**